frontend/src/routes/Video.tsx

```diff
@@ -423,12 +457,18 @@ const VideoPage: React.FC<Props> = ({ eventRef, realmRef, basePath }) => {
 
         <div css={{ height: 80 }} />
 
-        {event.series && <SeriesBlockFromSeries
-            basePath={basePath}
-            fragRef={event.series}
-            title={t("video.more-from-series", { series: event.series.title })}
-            activeEventId={event.id}
-        />}
+        {playlist
+            ? <PlaylistBlockFromPlaylist
+                basePath={basePath}
+                fragRef={playlist}
+            />
```

Comment:
It would kind of be nice to have a prefix for the block title here, like for series, no? "More from playlist" or sth like that? Or did you deliberately decide against that? I mean ok... but maybe do it differently from the series block in that you add an optional `prefix` prop or sth, so that you are not required to actually access `playlist.title` in this component, which won't be possible with the change suggested in my previous comment.

Reply:
nah I just automatically added missing props and didn't think about optional ones haha. "More from playlist" is good.